stylus/settings/palette.styl

```diff
@@ -155,6 +155,4 @@
     --primaryColorLighter #4B93F7
     --primaryColorLightest #9FC4FB
     --primaryContrastTextColor var(--white)
```

Comment:
Don't we have the same problem here ? Although I guess white ends up being white anyway…

Reply:
I think I was a bit imprecise before. CSS vars do not work for MUI when MUI has to make computations on it (making a color lighter or darker, to build all the variants of the color). I think we do not have the problem here because `primaryContrastTextColor` is not used as a base color for other colors, contrary to previously used `error.main` where MUI has to compute all shades of the `error` color.